### ISSUE TITLE
np.float deprecated

### ISSUE THREAD

**Issue by patrickmineault** [neurallatents/nlb_tools#12]
I ran into this error trying to replicate some of the results from the NDT paper:

```
Traceback (most recent call last):
  File "/Users/patrickmineault/Documents/unet-spikes/scripts/prep_nlb.py", line 96, in <module>
    main(args)
  File "/Users/patrickmineault/Documents/unet-spikes/scripts/prep_nlb.py", line 85, in main
    save_to_h5(h5_dict, h5_target, overwrite=True)
  File "/Users/patrickmineault/Documents/nlb_tools/nlb_tools/make_tensors.py", line 1296, in save_to_h5
    _save_h5_r(data_dict, h5file, dlen)
  File "/Users/patrickmineault/Documents/nlb_tools/nlb_tools/make_tensors.py", line 1361, in _save_h5_r
    dtype = f'float{dlen}' if val.dtype == np.float else f'int{dlen}' if val.dtype == np.int else val.dtype
  File "/Users/patrickmineault/miniconda3/envs/unet/lib/python3.9/site-packages/numpy/__init__.py", line 313, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```

Seems like a pretty straightforward fix: replace `np.float` with `float` and `np.int` with `int` in `make_tensors.py`.

**Comment by felixp8** [comment]
Thanks for raising the issue! We've had some discussion about how to best fix this in #11, but I've procrastinated on actually addressing it. I'll try to get to it either today or tomorrow.

**Comment by felixp8** [comment]
Should be resolved now with the merging of #11, so I'm closing this, but please reopen if it didn't actually fix your issues
